launch/motion_stack.launch

```diff
@@ -7,7 +7,7 @@
     </include>
```

Comment:
This file should also take a `platform` argument

Reply:
Fixed